furusiya/model/keys/callbacks.py

```diff
@@ -2,31 +2,45 @@
 from model.config import config
 
 
-def next_floor_callback():
-    Game.current_floor += 1
-
+def load_proper_area_map():
     Game.area_map.entities.remove(Game.player)
     if config.data.stallion.enabled:
         Game.area_map.entities.remove(Game.stallion)
 
     Game.area_map = Game.floors[Game.current_floor - 1]
 
-    Game.area_map.place_on_random_ground(Game.player)
+
+def switch_floor_callback(tile_to_spawn_player_around):
+    Game.area_map.place_around(Game.player, *tile_to_spawn_player_around)
     if config.data.stallion.enabled:
         if Game.stallion.is_mounted:
             Game.area_map.entities.append(Game.stallion)
             Game.stallion.x, Game.stallion.y = Game.player.x, Game.player.y
         else:
             Game.area_map.place_around(Game.stallion, Game.player.x, Game.player.y)
-
     Game.renderer.reset()
     Game.renderer.refresh_all()
 
 
+def next_floor_callback():
```

Comment:
I wouldn't use `callback` in names, it doesn't add any value. 

This entire thing seems to be coded strangely. Couldn't we just construct a `Stairs` game object and pass in a function or lambda to execute when the user goes up/down?